diff --git a/node_modules/@tiptap/pm/index-fixup.js b/node_modules/@tiptap/pm/index-fixup.js
new file mode 100644
index 0000000..7866e33
--- /dev/null
+++ b/node_modules/@tiptap/pm/index-fixup.js
@@ -0,0 +1 @@
+export * from "./state/dist/index.js"
diff --git a/node_modules/@tiptap/pm/index.js b/node_modules/@tiptap/pm/index.js
new file mode 100644
index 0000000..3266bd3
--- /dev/null
+++ b/node_modules/@tiptap/pm/index.js
@@ -0,0 +1,2 @@
+// Re-export from state as a default export point
+export * from './state/dist/index.js' 
\ No newline at end of file
